js/block.js

```diff
@@ -164,9 +164,9 @@ function Block(protoblock, blocks, overrideName) {
     // Internal function for creating cache.
     // Includes workaround for a race condition.
     this._createCache = function(callback, args) {
-        var that = this;
+        let that = this;
```

Comment:
This should be probably left as it is https://github.com/sugarlabs/musicblocks/pull/2201

Reply:
and other relevant places.

Reply:
Hi with this, I tested properly and used the ide tool to find usages of the variable, and I made sure I left `var` where it didn't suit `let`, but with all the `that = this` definitions in this file, the variable `that` was used only within that scope. Should I still revert to the use of `var`?

Reply:
@Tobenna-KA The issue being pointed out is that as we move towards arrow functions we won't need to do this arrangement. It is an obsolete workaround. So this line is anyway irrelevant.